### PR TITLE
OKTA-725768 Update name description for OIN and custom apps API

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Update name description for OIN and custom apps API
- **Is this PR related to a Monolith release?** No

### Preview:
https://preview-4855--reverent-murdock-829d24.netlify.app/docs/reference/api/apps/#add-an-oin-saml-2-0-authentication-app

### Resolves:

* [OKTA-725768](https://oktainc.atlassian.net/browse/OKTA-725768)

### CODE REVIEW

**Comment by vanngo-okta** [comment]
Support Eng approved on Slack ✅

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 1]
```diff
@@ -361,9 +361,13 @@ curl -v -X POST \
 }
 ```
 
-#### Add SAML 2.0 Authentication application
+#### Add an OIN SAML 2.0 Authentication app
+
+Adds a SAML 2.0 app instance for an app integration listed in the Okta Integration (OIN) catalog
+
+* The `name` field is required which identifies the OIN app definition for the instance you want to create. See [App name](#app-names).
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:368]:
```suggestion
* The `name` field is required, which identifies the OIN app definition for the instance that you want to create. See [App name](#app-names).
```

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 2]
```diff
@@ -361,9 +361,13 @@ curl -v -X POST \
 }
 ```
 
-#### Add SAML 2.0 Authentication application
+#### Add an OIN SAML 2.0 Authentication app
+
+Adds a SAML 2.0 app instance for an app integration listed in the Okta Integration (OIN) catalog
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:366]:
```suggestion
Adds a SAML 2.0 app instance for an app integration that's listed in the Okta Integration (OIN) catalog
```

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 3]
```diff
@@ -974,7 +984,14 @@ curl -v -X POST \
 
 #### Add custom SAML application
 
-Adds a SAML 2.0 application. This application is only available to the org that creates it.
+Adds a custom SAML 2.0 app. This application is only available to the org that creates it.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:987]:
```suggestion
Adds a custom SAML 2.0 app. This app is only available to the org that creates it.
```

File: packages/@okta/vuepress-site/docs/reference/api/apps/index.md [hunk 4]
```diff
@@ -7892,6 +7909,10 @@ Property details
 
 ##### App names
 
+The app name is the key that specifies the app definition schema required for custom or Okta Integration Network (OIN) apps. This field is required for all OIN and custom apps except for the custom SAML 2.0 or SWA app. For a custom SAML 2.0 or SWA app, the app name is autogenerated when you create the app object.
```
Comment (barbaravo-okta) [inline packages/@okta/vuepress-site/docs/reference/api/apps/index.md:7912]:
```suggestion
The app name is the key that specifies the app definition schema that's required for custom or Okta Integration Network (OIN) apps. This field is required for all OIN and custom apps except for the custom SAML 2.0 or SWA app. For a custom SAML 2.0 or SWA app, the app name is autogenerated when you create the app object.
```

**Review by barbaravo-okta — DISMISSED** [review]

**Review by barbaravo-okta — APPROVED** [review]

**Pull request merged**